tracking/migrations/0001_initial.py

```diff
@@ -29,4 +29,18 @@ class Migration(migrations.Migration):
                 'get_latest_by': 'timestamp',
             },
         ),
+        migrations.CreateModel(
```

Comment:
This migration should be added as a new file rather than existing one. You should be able to generate this from Django with ./manage.py makemigrations.

This should also be in seumich/migrations if the model is moved. Over there it would be **0004_...**

This would allow the migration to run on dev/prod.

Reply:
I need to talk with you tomorrow on how to generate the migration file.